dev-client/src/model/color/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+export type RGBA = [number, number, number, number];
+export type RGB = [number, number, number];
```

Comment:
question: is it worth creating a RGBValue type that is a subset of number, (i.e. 0-255)? https://stackoverflow.com/questions/39494689/is-it-possible-to-restrict-number-to-a-certain-range

Reply:
no i don't think so:
- we're never manually constructing RGBs so there wouldn't be any situation where that could actually validate anything
- it would make the definitions much more difficult to understand
- it could be a typechecker performance papercut which could add up if we did things like that too often